src/compiler/checker.ts

```diff
@@ -8475,11 +8467,11 @@ namespace ts {
                         return type;
                     }
                     const doubleEquals = operator === SyntaxKind.EqualsEqualsToken || operator === SyntaxKind.ExclamationEqualsToken;
-                    const facts = doubleEquals ?
-                        assumeTrue ? TypeFacts.EQUndefinedOrNull : TypeFacts.NEUndefinedOrNull :
-                        value.kind === SyntaxKind.NullKeyword ?
-                            assumeTrue ? TypeFacts.EQNull : TypeFacts.NENull :
-                            assumeTrue ? TypeFacts.EQUndefined : TypeFacts.NEUndefined;
+                    const facts = doubleEquals
+                        ? assumeTrue ? TypeFacts.EQUndefinedOrNull : TypeFacts.NEUndefinedOrNull
+                        : value.kind === SyntaxKind.NullKeyword
+                            ? assumeTrue ? TypeFacts.EQNull : TypeFacts.NENull
+                            : assumeTrue ? TypeFacts.EQUndefined : TypeFacts.NEUndefined;

```

Comment:
I can change it back. I primarily made the change to work around an issue with our .tmLanguage syntax coloring as the existing formatting resulted in syntax coloring being disabled/incorrect for the next 30 lines or so in VS Code.